hawkular-wildfly-agent/src/main/resources/schema/hawkular-agent-monitor-subsystem.xsd

```diff
@@ -394,14 +394,16 @@
     <xs:attribute name="name"          type="xs:string" use="required"/>
     <xs:attribute name="type"          type="operationParamTypeType" default="string"/>
     <xs:attribute name="default-value" type="xs:string" />
+    <xs:attribute name="required"      type="xs:boolean" default="false" use="optional"/>
```

Comment:
to make the xsd less noisy, we do not specify "use="optional"" on any xs:attribute since that is the default. Remote "use" attribute here since optional is inferred.